LOAN/RealEstateLoans/ConstructionLoans.rdf

```diff
@@ -41,140 +37,89 @@
-		<rdfs:label>manufactured</rdfs:label>
-		<skos:definition>a factory-built dwelling built in compliance with the Federal Manufactured Home Construction and Safety Standards in effect at the time the home was manufactured as evidenced by the HUD label</skos:definition>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-reln-cnst;ConstructionType_MobileHome">
-		<rdf:type rdf:resource="&fibo-loan-reln-cnst;ConstructionType"/>
-		<rdfs:label>mobile home</rdfs:label>
-		<skos:definition>a dwelling unit constructed on a base frame which features wheels and axles to be used in transporting home from place to place</skos:definition>
-		<cmns-av:explanatoryNote>It does not meet HUD code.</cmns-av:explanatoryNote>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-reln-cnst;ConstructionType_Modular">
-		<rdf:type rdf:resource="&fibo-loan-reln-cnst;ConstructionType"/>
-		<rdfs:label>modular</rdfs:label>
-		<skos:definition>a factory-built dwelling not on a permanent chassis</skos:definition>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-reln-cnst;ConstructionType_OnFrameModular">
-		<rdf:type rdf:resource="&fibo-loan-reln-cnst;ConstructionType"/>
-		<rdfs:label>on frame modular</rdfs:label>
-		<skos:definition>a factory built dwelling on a permanent chassis which does not have a HUD label</skos:definition>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-reln-cnst;ConstructionType_SiteBuilt">
-		<rdf:type rdf:resource="&fibo-loan-reln-cnst;ConstructionType"/>
-		<rdfs:label>site built</rdfs:label>
-		<skos:definition>describes construction process, indicating that most elements are created at the homes permanent site. May include some prefabricated components</skos:definition>
-	</owl:NamedIndividual>
-	
 	<owl:ObjectProperty rdf:about="&fibo-loan-reln-cnst;hasMaximumAllowedBalance">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fbc-dae-dbt;hasAvailableAmount"/>
+		<rdfs:subPropertyOf rdf:resource="&fibo-fbc-pas-caa;hasBalance"/>
 		<rdfs:label xml:lang="en">has maximum allowed balance</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-loan-reln-cnst;ConstructionLoanContract"/>
+		<rdfs:domain>
+			<owl:Class>
+				<owl:unionOf rdf:parseType="Collection">
+					<rdf:Description rdf:about="&fibo-loan-reln-cnst;ConstructionLoan">
+					</rdf:Description>
+					<rdf:Description rdf:about="&fibo-fbc-dae-dbt;SubFacility">
+					</rdf:Description>
+				</owl:unionOf>
+			</owl:Class>
+		</rdfs:domain>
 		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
-		<skos:definition xml:lang="en">Maximum balance - For loans with flexible re-draw facilities, the maximum balance that could potentially be outstanding based on a credit limit</skos:definition>
+		<skos:definition xml:lang="en">for loans with flexible re-draw facilities, the maximum balance that may potentially be outstanding at any time below the credit limit</skos:definition>
```

Comment:
"below the credit limit" makes this ambiguous. Does it mean "outstanding at any time, that is still below the credit limit"? Can it be equal?
Why do we even need both this property and hasCreditLimit?

Reply:
That came from the details related to reporting - we can take a deeper dive when we focus more on construction lending. There are more references to incorporate before we release this.